Initium-Odp/src/com/universeprojects/miniup/server/controllers/ViewItemController.java

```diff
@@ -683,6 +683,16 @@ else if(ownerHtmlSpecified == false && GameUtils.enumEquals(scriptType, ODPDBAcc
 		if (field!=null && field.toString().trim().equals("")==false)
 			itemMap.put("description", field.toString());
 		
+		//If we have explicitely allowed charges to be visible on this item, display them.
+		Boolean chargesVisible = (boolean) item.getProperty("chargesVisible");
+		if(chargesVisible == null) chargesVisible = false;
+		if(chargesVisible) {
+			field = item.getProperty("charges");

```

Comment:
I find it odd that you're double indenting in all your code, can we use single indents? an indent should be done via the TAB button and it should be no more than 4 spaces wide per indent.